packages/components/src/components/my-position/my-position.tsx

```diff
@@ -0,0 +1,402 @@
+import { Component, Host, JSX, Prop, Event, EventEmitter, State, h } from '@stencil/core';
+import { UAParser } from 'ua-parser-js';
+import merge from 'deepmerge';
+import midtColors from '@mapsindoors/midt/tokens/color.json';
+import midtOpacity from '@mapsindoors/midt/tokens/opacity.json';
+import { getNestedValue } from '../../utils/utils';
+
+enum PositionStateTypes {
+    POSITION_UNKNOWN = 'POSITION_UNKNOWN',
+    POSITION_REQUESTING = 'POSITION_REQUESTING',
+    POSITION_INACCURATE = 'POSITION_INACCURATE',
+    POSITION_KNOWN = 'POSITION_KNOWN',
+    POSITION_CENTERED = 'POSITION_CENTERED',
+    POSITION_TRACKED = 'POSITION_TRACKED',
+    POSITION_UNTRACKED = 'POSITION_UNTRACKED'
+}
+
+enum DeviceType {
+    Phone = 'mobile',
+    Tablet = 'tablet'
+}
+
+@Component({
+    tag: 'mi-my-position',
+    styleUrl: 'my-position.scss',
+    shadow: true,
```

Comment:
Should this be open for styling from the outside just like we did with the Floor Selector?

Reply:
@ammapspeople I talked to @markhougaard and we agreed to have it open, yes